Added info that features will only be available as beta initially. (#3091)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-53-0-new-data-point-explorer-and-data-points-graph.md

---
date: ""

> title: Enhanced data point explorer and data point graph widget

Reviewer: ```suggestion
title: Enhanced data point explorer and data point graph widgets
```

ticket: MTM-61539
version: 1021.53.0
---

> To enhance data analysis and visualization within the {{< product-c8y-iot >}} platform, the data point explorer and data point graph have been upgraded with new capabilities and a modernized architecture. Both features have been migrated from AngularJS to Angular, bringing improved performance, a refreshed interface, and expanded configuration options. Initially, these features will not be included by default and can only be added as beta features.

Reviewer 2: should we add info how customers can access beta features?

Reviewer 3: That is a good question... maybe we need to explain this more in detail?

Author: @jdre-c8y  @janhommes  Done, added improved description.

> To enhance data analysis and visualization within the {{< product-c8y-iot >}} platform, the data point explorer and data point graph have been upgraded with new capabilities and a modernized architecture. Both features have been migrated from AngularJS to Angular, bringing improved performance, a refreshed interface, and expanded configuration options. Initially, these features will not be included by default and can only be added as beta features.

Author: ```suggestion
To enhance data analysis and visualization within the {{< product-c8y-iot >}} platform, the data point explorer and data point graph have been upgraded with new capabilities and a modernized architecture. Both features have been migrated from AngularJS to Angular, bringing improved performance, a refreshed interface, and expanded configuration options. Initially, these features will not be included by default and can only be installed as additional plugins. To install them, navigate to your custom application, click on "Install Plugins," and search for "Data Point Explorer" and "Data Point Graph." .
```

> To enhance data analysis and visualization within the {{< product-c8y-iot >}} platform, the data point explorer and data point graph have been upgraded with new capabilities and a modernized architecture. Both features have been migrated from AngularJS to Angular, bringing improved performance, a refreshed interface, and expanded configuration options. Initially, these features will not be included by default and can only be installed as additional plugins. To install them, navigate to your custom application, click **Install plugins**, and search for "Data point explorer" and "Data point graph." 

Reviewer: ```suggestion
To enhance data analysis and visualization within the {{< product-c8y-iot >}} platform, the data point explorer and data point graph have been upgraded with new capabilities and a modernized architecture. Both features have been migrated from AngularJS to Angular, bringing improved performance, a refreshed interface, and expanded configuration options. Initially, these features will not be included by default and can only be installed as additional plugins. To install them, navigate to your custom application, click **Install plugins**, and search for "Data point explorer" and "Data point graph".
```

> To enhance data analysis and visualization within the {{< product-c8y-iot >}} platform, the data point explorer and data point graph have been upgraded with new capabilities and a modernized architecture. Both features have been migrated from AngularJS to Angular, bringing improved performance, a refreshed interface, and expanded configuration options. Initially, these features will not be included by default and can only be installed as additional plugins. To install them, navigate to your custom application, click on "Install Plugins," and search for "Data Point Explorer" and "Data Point Graph." .

Reviewer: ```suggestion
To enhance data analysis and visualization within the {{< product-c8y-iot >}} platform, the data point explorer and data point graph have been upgraded with new capabilities and a modernized architecture. Both features have been migrated from AngularJS to Angular, bringing improved performance, a refreshed interface, and expanded configuration options. Initially, these features will not be included by default and can only be installed as additional plugins. To install them, navigate to your custom application, click **Install plugins**, and search for "Data point explorer" and "Data point graph." 
```
